app/views/overview.html

```diff
@@ -122,8 +122,8 @@ <h2>
                   <div class="component-label">Application</div>
                   <span ng-bind-html="app | highlightKeywords : overview.state.filterKeywords"></span>
                 </h2>
-                <div ng-if="route = overview.bestRouteByApp[app]" class="pull-right">
-                  <h3 class="overview-route">
+                <div class="overview-routes" ng-if="overview.routesToDisplayByApp[app].length > 0">
+                  <h3 class="overview-route" ng-repeat="route in overview.routesToDisplayByApp[app] track by $index">
```

Comment:
Use `route | uid` for `track by`. UID is a better identifier for that route than the index in the array since things might get reordered.

```html
<h3 class="overview-route" ng-repeat="route in overview.routesToDisplayByApp[app] track by (route | uid)">
```